src/components/coach/CoachQuickTrackModal.tsx

```
@@ -0,0 +1,454 @@
+              Back
+            </Button>
+            <Button onClick={() => setStep('confirm')} className="gap-2">
+              Next
+              <ArrowRight className="w-4 h-4" />
+            </Button>
+          </>
+        );
+      
+      case 'confirm':
+        const canStart = playerValidation?.isValid && !loading && !validationLoading;
+        return (
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)